client/my-sites/plans-v2/plans-column/index.tsx

```diff
@@ -15,13 +15,19 @@ import {
 	OPTIONS_JETPACK_SECURITY,
 	OPTIONS_JETPACK_SECURITY_MONTHLY,
 } from '../constants';
-import { PLAN_JETPACK_FREE } from 'lib/plans/constants';
+import {
+	PLAN_JETPACK_FREE,
+	PLAN_JETPACK_SECURITY_DAILY,
+	PLAN_JETPACK_SECURITY_DAILY_MONTHLY,
+} from 'lib/plans/constants';
+import { isUpgradeable, getRealtimeFromDaily } from 'my-sites/plans-v2/utils';
```

Comment:
There is already a utils import above on line 11, and constants right afterward. The code should stick to one block of imports from the same file.